src/Shapefile.jl

```diff
@@ -14,23 +19,26 @@ struct Interval
     right::Float64
 end
 
-struct Point <: GeoInterface.AbstractPoint
-    x::Float64
-    y::Float64
-end
+const Point = GB.Point{2, Float64}
```

Comment:
should this be `Pointe0` instead? (a la Pointf0)

Reply:
Ha yeah, though then it should also include the 2 I guess.

I think for now it is easiest to keep the names the same as the original structs though. They won't be exported anyway, only used as internal aliases.

Reply:
Yeah, if we do anything, this shoulg go into GeometryBascs!

Reply:
Although, `Point0` would be super ambigious.. I guess we'd rather go for something like `Point2d` for double... Although that's also ambigious, but at least doesnt look as weird^^